Expensify/App
# Add latest bank + paycheck icons

### Details

Replaces bank & paycheck icons with updated versions 👍 

cc @marcaaron @shawnborton 

### Fixed Issues

Fixes https://github.com/Expensify/Expensify/issues/167404

### Tests
1. Go to `/bank-account` route
2. Verify icons look gray (on Web if you hover over them, they should turn black)

### QA Steps
Same as above

### Tested On

- [ ] Desktop - N/A (can't get to `/bank-account` route yet)
- [ ] iOS - N/A (can't get to `/bank-account` route yet)
- [ ] Android - N/A (can't get to `/bank-account` route yet)

### Screenshots


#### Web

https://user-images.githubusercontent.com/3885503/122227628-57a99280-ceb7-11eb-937b-1b78ebf4908f.mov

#### Mobile Web

![Screen Shot 2021-06-16 at 3 21 37 PM](https://user-images.githubusercontent.com/3885503/122227612-537d7500-ceb7-11eb-84a7-a782a356d080.png)

#### Desktop

N/A (can't get to `/bank-account` route yet)

#### iOS

N/A (can't get to `/bank-account` route yet)

#### Android

N/A (can't get to `/bank-account` route yet)
